src/jdk.jdwp.agent/unix/native/libjdwp/exec_md.c

```diff
@@ -62,7 +62,7 @@ static char *skipNonWhitespace(char *p) {
 // input/output/error file descriptors will not be closed
 // by this function. This function returns 0 on failure
 // and 1 on success.
-int
+static int
```

Comment:
I think you should also make forkedChildProcess static. It was added at the same time.

Reply:
Updated.